src/MicrosoftTeams.schema.json

```diff
@@ -279,8 +298,10 @@
         "validDomains": {
             "type": "array",
             "description": "A list of valid domains from which the tabs expect to load any content. Domain listings can include wildcards, for example `*.example.com`. If your tab configuration or content UI needs to navigate to any other domain besides the one use for tab configuration, that domain must be specified here.",
+            "maxItems": 16,
             "items": {
-                "type": "string"
+                "type": "string",
+                "maxLength": 2048
             }
         }
     },
```

Comment:
Please add "name" and "description" to the list of required attributes below. I think we accidentally got rid of it when we went from v0.3 --> v0.4 --> v0.5.